samples/api-client/mqtt_example/cloudiot_mqtt_example.py

```diff
@@ -111,33 +117,44 @@ def on_disconnect(unused_client, unused_userdata, rc):
 
 def on_publish(unused_client, unused_userdata, unused_mid):
     """Paho callback when a message is sent to the broker."""
-    print('on_publish')
+    print("on_publish")
 
 
 def on_message(unused_client, unused_userdata, message):
     """Callback when the device receives a message on a subscription."""
-    payload = str(message.payload.decode('utf-8'))
-    print('Received message \'{}\' on topic \'{}\' with Qos {}'.format(
-            payload, message.topic, str(message.qos)))
+    payload = str(message.payload.decode("utf-8"))
+    print(
+        "Received message '{}' on topic '{}' with Qos {}".format(
+            payload, message.topic, str(message.qos)
+        )
+    )
 
 
 def get_client(
-        project_id, cloud_region, registry_id, device_id, private_key_file,
-        algorithm, ca_certs, mqtt_bridge_hostname, mqtt_bridge_port):
+    project_id,
+    cloud_region,
+    registry_id,
+    device_id,
+    private_key_file,
+    algorithm,
+    ca_certs,
+    mqtt_bridge_hostname,
+    mqtt_bridge_port,
+):
     """Create our MQTT client. The client_id is a unique string that identifies
     this device. For Google Cloud IoT Core, it must be in the format below."""
```

Comment:
nit: if we're following the Google Docstring style, it should be formatted more like

```python
"""Create our MQTT client.

The client_id is a unique string that identifies this device. For 
Google Cloud IoT Core, it must be in the format below.
"""
```
see https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings.

Reply:
done